kubernetes/rt.yml.erb

```diff
@@ -40,6 +40,22 @@ spec:
           env:
             - name: SERVER_NAME
               value: rt.ocf.berkeley.edu
+          livenessProbe:
+            exec:
+              command:
+              - sh
+              - '-c'
+              - '/opt/rt/healthcheck $SERVICE_SERVICE_HOST $SERVICE_SERVICE_PORT'
+            initialDelaySeconds: 5
```

Comment:
If RT usually takes more than 5 seconds to start, we probably want to bump this up. My understanding is that the initial delay should be longer than we would expect normal startup time to take.

Reply:
Sure, I'll bump it to 10 seconds.